app/services/support_interface/candidate_journey_tracker.rb

```diff
@@ -143,7 +143,16 @@ def all_references
     end
 
     def received_references
-      @received_references ||= all_references.select(&:feedback_provided?)
+      @received_references ||=
+        begin
+          references = all_references.unscoped.feedback_provided
+
+          if references.any? { |ref| ref.feedback_provided_at.blank? } # an error state caused by bad data
+            [] # show nothing about references in the export so that the application can be easily ignored/filtered out
```

Comment:
We should assess if it's possible to fix the data (I'll write up a card for this), but in the meantime this seems like the simplest thing to do. Milan is already filtering out applications with blank `feedback_provided_at`.